includes/admin/class.llms.admin.user.custom.fields.php

```diff
@@ -104,72 +106,15 @@ public function add_errors( &$errors, $update, $user ) {
 	 *
 	 * @since 2.7.0
 	 * @since 3.13.0 Unknown.
+	 * @since [version] Removed LLMS core fields.
 	 *
 	 * @return array
 	 */
 	public function get_fields() {
 
 		$fields = apply_filters(
```

Comment:
Use `apply_filters_deprecated()` (with a filter doc)

Reply:
Okay yea that sounds better

Reply:
I was wrong, I didn't realize that apply_filters_deprecated actually checks whether the hook as at leas one filter callback added. So let's go with `apply_filters_deprecated` as you suggested.